Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/misc.yml

```
@@ -0,0 +1,17 @@
+- type: entity  #Delta-V realistic PPE apron for chemistry
```

```suggestion
- type: entity
```
No need for a change marker since it is in a DeltaV file.